server/auth/types/basic/routes.ts

```diff
@@ -53,26 +51,33 @@ export class BasicAuthRoutes {
         },
       },
       async (context, request, response) => {
-        const forbiddenUsernames = this.config.auth.forbidden_usernames;
+        const forbiddenUsernames: string[] = this.config.auth.forbidden_usernames;
         if (forbiddenUsernames.indexOf(request.body.username) > -1) {
-          throw new Error('Invalid username or password'); // Cannot login using forbidden user name.
+          context.security_plugin.logger.error(
+            `Denied login for forbidden username ${request.body.username}`
+          );
+          return response.badRequest({
+            // Cannot login using forbidden user name.
+            body: 'Invalid username or password',
```

Comment:
Shall we call it "Forbidden username" to distinguish this case from where the credentials are wrong?